packages/trader/src/AppV2/Utils/trade-types-utils.tsx

```diff
@@ -0,0 +1,205 @@
+import React from 'react';
+import { Localize, localize } from '@deriv/translations';
+import { Text } from '@deriv-com/quill-ui';
+import VideoFragment from 'AppV2/Components/Guide/Description/video-fragment';
+import { DESCRIPTION_VIDEO_ID } from 'Modules/Trading/Helpers/video-config';
+
+type TDtraderVideoUrl = {
+    [key: string]: TVideoVariants;
+};
+
+type TVideoVariants = {
+    dark: string;
+    light: string;
+};
+
+export const CONTRACT_LIST = {
+    ACCUMULATORS: 'Accumulators',
+    VANILLAS: 'Vanillas',
+    TURBOS: 'Turbos',
+    MULTIPLIERS: 'Multipliers',
+    RISE_FALL: 'Rise/Fall',
+    HIGHER_LOWER: 'Higher/Lower',
+    TOUCH_NO_TOUCH: 'Touch/No Touch',
+    MATCHES_DIFFERS: 'Matches/Differs',
+    EVEN_ODD: 'Even/Odd',
+    OVER_UNDER: 'Over/Under',
+};
+
+export const AVAILABLE_CONTRACTS = [
+    { tradeType: <Localize i18n_default_text='Accumulators' />, id: CONTRACT_LIST.ACCUMULATORS },
+    { tradeType: <Localize i18n_default_text='Vanillas' />, id: CONTRACT_LIST.VANILLAS },
+    { tradeType: <Localize i18n_default_text='Turbos' />, id: CONTRACT_LIST.TURBOS },
+    { tradeType: <Localize i18n_default_text='Multipliers' />, id: CONTRACT_LIST.MULTIPLIERS },
+    { tradeType: <Localize i18n_default_text='Rise/Fall' />, id: CONTRACT_LIST.RISE_FALL },
+    { tradeType: <Localize i18n_default_text='Higher/Lower' />, id: CONTRACT_LIST.HIGHER_LOWER },
+    { tradeType: <Localize i18n_default_text='Touch/No Touch' />, id: CONTRACT_LIST.TOUCH_NO_TOUCH },
+    { tradeType: <Localize i18n_default_text='Matches/Differs' />, id: CONTRACT_LIST.MATCHES_DIFFERS },
+    { tradeType: <Localize i18n_default_text='Even/Odd' />, id: CONTRACT_LIST.EVEN_ODD },
+    { tradeType: <Localize i18n_default_text='Over/Under' />, id: CONTRACT_LIST.OVER_UNDER },
+];
+
+export const TERM = {
+    BARRIER: localize('Barrier'),
+    CONTRACT_VALUE: localize('Contract value'),
+    DEAL_CANCELLATION: localize('Deal cancellation'),
+    ENTRY_SPOT: localize('Entry spot'),
+    EXIT_SPOT: localize('Exit spot'),
+    EXPIRY: localize('Expiry'),
+    FINAL_PRICE: localize('Final price'),
+    GROWTH_RATE: localize('Growth rate'),
+    PAYOUT: localize('Payout'),
+    PAYOUT_PER_POINT: localize('Payout per point'),
+    PREVIOUS_SPOT_PRICE: localize('Previous spot price'),
+    RANGE: localize('Range'),
+    SLIPPAGE_RISK: localize('Slippage risk'),
+    STOP_OUT_LEVEL: localize('Stop out level'),
+    STOP_LOSS: localize('Stop loss'),
+    STRIKE_PRICE: localize('Strike price'),
+    TAKE_PROFIT: localize('Take profit'),
+};
+
+const DEFINITION = {
```

Comment:
Can we move this to a separate file? This file is getting too big.

Reply:
Sure, will split up